src/bitcoin-cli.cpp

```diff
@@ -1101,6 +1101,68 @@ static UniValue GetNewAddress()
     return ConnectAndCallRPC(&rh, "getnewaddress", /* args=*/{}, wallet_name);
 }
 
+/**
+ * IsExclusivelyCliCommand checks if a string is a cli-command that needs to run exclusively
+ * @param[in] cliCommand  Reference to const string cli-command.
+ * @returns true if cliCommand it's included in the set of the cli-commands.
+ */
+bool IsExclusivelyCliCommand(const std::string cliCommand)
+{
+    // Initialize set with cli-commands that run exclusively
+    const std::set<std::string> cliCommands{"-generate", "-netinfo", "-addrinfo", "-getinfo"};
+
+    // return if element was found or not
+    return (cliCommands.find(cliCommand) != cliCommands.end());
```

Comment:
Here we are using `std::set::find` and below on line 1141 we are using `std::set::count` to determine if an element already exists. We should use a consistent method for each. I'm not sure if C++20 is supported in this codebase yet, but we should use `std::set::contains` if it is.

Reply:
It makes sense, I'll use `std::set::find` instead of `std::set::count` (L#1141).

Regarding C++20. it's [not enabled](https://github.com/bitcoin/bitcoin/blob/master/configure.ac#L99) yet but perhaps will be soon (#28349). If this PR doesn't get merged first I'll update the code as you suggested so I'm leaving this conversation as unresolved, otherwise I'll do a follow-up.

Reply:
@andrewtoth, now I've included `std::set::contains` as [you](https://github.com/bitcoin/bitcoin/pull/26990#discussion_r1383787269) and @jonatack [recommended](https://github.com/bitcoin/bitcoin/pull/26990#discussion_r1476755003).